### PR TITLE
fix(SUP-20644): VTT not working on iOS when KS is added

### PR DESCRIPTION
**Pull request by shalommeoded**
When KS is added (KMS for example) to the VTT call the call is sent wrong and a result broken.

Currently:
http://cdnapi.kaltura.com/api_v3/index.php/service/caption_captionasset/action/serveWebVTT/captionAssetId/0_svo0de7e/segmentIndex/-1/version/2/**captions.vtt**/ks/NjdlOTkwMWFkYzNlNTNiMjgyZTczMTBjNWE1MGQ2OGU2OTcyMDExYnw1NDIxMzE7NTQyMTMxOzE1Nzk1MzEwNzY7MjsxNTc5NDQ0Njc2LjY5NDY7YWRtaW47ZGlzYWsdfsdfbGVtZW50Ozs=

With the fix:
http://cdnapi.kaltura.com/api_v3/index.php/service/caption_captionasset/action/serveWebVTT/captionAssetId/0_svo0de7e/segmentIndex/-1/version/2/ks/NjdlOTkwMWFkYzNlNTNiMjgyZTczMTBjNWE1MGQ2OGU2OTcyMDExYnw1NDIxMzE7NTQyMTMxOzE1Nzk1MzEwNzY7MjsxNTc5NDQ0Njc2LjY5NDY7YWRtaW47ZGlzYWJssdfsdfZW50Ozs=/captions.vtt

Basically I moved the captions.vtt of the captions to the end.

### CODE REVIEW

File: modules/KalturaSupport/resources/mw.ClosedCaptions.js [hunk 1]
```diff
@@ -577,8 +577,8 @@
 				captionsSrc = mw.getConfig('Kaltura.ServiceUrl') +
 							"/api_v3/index.php/service/caption_captionasset/action/serveWebVTT/captionAssetId/" +
 							dbTextSource.id +
-							"/segmentIndex/-1/version/2/captions.vtt";
-				captionsSrc += ks ? '/ks/' + ks : '';
+							"/segmentIndex/-1/version/2";
+				captionsSrc += ks ? "/ks/" + ks + "/captions.vtt" : "/captions.vtt";
```
Comment (yairans) [inline modules/KalturaSupport/resources/mw.ClosedCaptions.js]:
```suggestion
				captionsSrc += ks ? "/ks/" + ks : "" + "/captions.vtt";
```

Comment (shalommeoded) [reply modules/KalturaSupport/resources/mw.ClosedCaptions.js]: @yairans 
We need the .vtt extension (/captions.vtt) at the end in order to recognize it as a VTT file.

Will work:

`http://cdnapi.kaltura.com/api_v3/index.php/service/caption_captionasset/action/serveWebVTT/captionAssetId/0_svo0de7e/segmentIndex/-1/version/2/ks/NjdlOTkwMWFkYzNlNTNiMjgyZTczMTBjNWE1MGQ2OGU2OTcyMDExYnw1NDIxMzE7NTQyMTMxOsadfdsdfsdfzY7MjsxNTc5NDQ0Njc2LjY5NDY7YWRtaW47ZGlzYWJsZWVudGl0bGVtZW50Ozs=/captions.vtt`

Will not work:
http://cdnapi.kaltura.com/api_v3/index.php/service/caption_captionasset/action/serveWebVTT/captionAssetId/0_svo0de7e/segmentIndex/-1/version/2/ks/NjdlOTkwMWFkYzNlNTNiMjgyZTczMTBjNWE1MGQ2OGU2OTcyMDExYnw1NDIxMzE7NTQyMTMxOsadfdsdfsdfzY7MjsxNTc5NDQ0Njc2LjY5NDY7YWRtaW47ZGlzYWJsZWVudGl0bGVtZW50Ozs=
